.changelog/3845.breaking.1.md

```diff
@@ -0,0 +1,3 @@
+go/common/entity: Rename `LatestEntityDescriptorVersion`
```

Comment:
Probably not a breaking change

Reply:
Technically, it was public and people could use it. I checked and there are [2 public imports of `go/common/entity` at the moment](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/common/entity).

But yeah, the impact should be really small or non-existing. I did it because we are just before a new release :slightly_smiling_face:.

Reply:
Ah yeah makes sense being nice and keeping it under breaking then :)